fireworks_poe_bot/plugin.py

```diff
@@ -0,0 +1,38 @@
+from abc import ABC, abstractmethod
+from typing import Dict, List
+
+
+class LoggingPlugin(ABC):
```

Comment:
A more standard way would be to do it through python logging handlers, e.g., https://stackoverflow.com/questions/3118059/how-to-write-custom-python-logging-handler

Reply:
We already use this https://github.com/fw-ai/fireworks_poe_bot/blob/main/fireworks_poe_bot/logging.py

And this is orthogonal. We want to add GCP structured logging rather than redirecting arbitrary line-based logging